src/main/scala/com/gu/restaurant_review_parser/ParsedRestaurantReview.scala

```diff
@@ -0,0 +1,49 @@
+package com.gu.restaurant_review_parser
+
+import java.time.OffsetDateTime
+
+case class WebAddress(value: String) extends AnyVal
```

Comment:
@tsop14 check out this guy I know called Chris: http://slides.com/cb372/real-world-func-scala-lambdaworld-2016#/15 ;)